wordpress-mobile/WordPress-Android
# Ensure release notes changes are localizable

**To test:**
1) Make a change to `release_notes.txt` in this branch. Don't update `PlayStoreStrings.po`.
2) Danger should provide a warning. 
3) Delete your commit, and force-push.
4) Make a change to `release_notes.txt` in this branch. This time, update `PlayStoreStrings.po` by running `bundle exec fastlane update_ps_strings version:11.6`.
5) Danger should provide no warnings.
6) Delete your commit, and force-push.

Update release notes:
